app/test_split_integer.py

```diff
@@ -2,20 +2,21 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(value=32, number_of_parts=6)) == 32
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert split_integer(value=16, number_of_parts=4) == [4, 4, 4, 4]
```

Comment:
Add test when the value is not equally divided by number_of_parts

Reply:
I need to change this test? Add new assert? Write new test?